<launch>
  <arg name="robot" default="pr2"/> <!-- pr2, baxter, fetch -->
  <arg name="input_image" default="/kinect_head/rgb/image_rect_color"/>
  <arg name="camera_info" default="/kinect_head/rgb/camera_info"/>
  <arg name="joint_states" default="/joint_states"/>
  <arg name="robot_urdf" default="$(find virtual_robot_ros)/data/urdf/$(arg robot).urdf"/>
  <arg name="base_frame" default="base_footprint"/>

  <!-- whole robot body render -->
  <node name="robot_render_node" pkg="virtual_robot_ros" type="robot_render_node.py" output="screen">
    <remap from="~input_image" to="$(arg input_image)" />
    <remap from="~camera_info" to="$(arg camera_info)" />
    <remap from="~joint_states" to="$(arg joint_states)" />
    <rosparam subst_value="true" >
        robot_urdf: $(arg robot_urdf)
        base_frame: $(arg base_frame)
    </rosparam>
  </node>

</launch>
